### PR TITLE
switch to new Azure release workflows

### PR DESCRIPTION
**Pull request by davidgamero**
the old release process is no longer viable, and a new reusable workflow is provided in https://www.github.com/Azure/action-release-workflows

### CODE REVIEW

**Review by erinborders — APPROVED** [review]

File: src/run.test.ts [hunk 1]
```diff
@@ -205,7 +205,4 @@ describe('Set context', () => {
       expect(fs.chmodSync).toBeCalledWith(kubeconfigPath, '600')
       expect(core.exportVariable).toBeCalledWith('KUBECONFIG', kubeconfigPath)
    })
-
```
Comment (jaiveerk) [inline src/run.test.ts:208]: nit: newlines

**Review by jaiveerk — CHANGES_REQUESTED** [review]

File: src/run.test.ts (hunk 1 above)
Comment (davidgamero) [reply src/run.test.ts:208]: that got added by the prettier formatter i can try to remove it

Comment (davidgamero) [reply src/run.test.ts:208]: yeah those are out of our format spec we can't add them back without an override

Comment (jaiveerk) [reply src/run.test.ts:208]: ah ok nw nw

**Review by jaiveerk — APPROVED** [review]

**Pull request merged**